boards/xtensa/heltec_wireless_stick_lite_v3/doc/index.rst

```diff
@@ -0,0 +1,306 @@
+.. heltec_wireless_stick_lite_v3:
+
+HelTec Wireless Stick Lite (V3)
+###############################
+
+Overview
+********
+
+HelTec Wireless Stick Lite (V3) is a development board with Wi-Fi, Bluetooth and LoRa support. It is designed and produced by HelTec Automation(TM). [1]_
+
+.. figure:: heltec_wireless_stick_lite_v3.jpg
+   :width: 400px
+   :align: center
+   :alt: HelTec Wireless Stick Lite (V3)
+
+   HelTec Wireless Stick Lite (V3) (Credit: Chengdu HelTec Automation Technology Co., Ltd.)
+
+Hardware
+********
+
+The main hardware features are:
+
+- ESP32-S3FN8 low-power MCU-based SoC (dual-core Xtensa® 32-bit LX7 microprocessor, five stage pipeline rack Structure, main frequency up to 240 MHz).
+- Semtech SX1262 LoRa node chip
+- Type-C USB interface with a complete voltage regulator, ESD protection, short circuit protection, RF shielding, and other protection measures (note: you need an USB-A to USB-C cable if you want power-up the board from USB).
+- Onboard SH1.25-2 battery interface, integrated lithium battery management system (charge and discharge management, overcharge protection, battery power detection, USB / battery power automatic switching).
+- Integrated WiFi and Bluetooth interfaces with 2.4GHz metal spring antenna and reserved IPEX (U.FL) interface for LoRa use.
+- Integrated CP2102 USB to serial port chip, convenient for program downloading, debugging information printing.
+- Good RF circuit design and low-power design.
+
+Supported Features
+==================
+- LoRa via SPI
+- UART0 (USB Serial via CP2102)
+- UART1
+- I2C
+- CAN (optional, need to enable)
+- PWM LED
+- User Switch / Button
+
+Connections and IOs
+===================
+
+.. figure:: heltec_wireless_stick_lite_v3_pinout.jpg
+   :width: 600px
+   :align: center
+   :alt: HelTec Wireless Stick Lite (V3) Pinout
+
+   Pinout (Credit: Chengdu HelTec Automation Technology Co., Ltd.)
+
+.. table:: HelTec Wireless Stick Lite (V3) Pinout
+   :widths: auto
+
+   +--------+---------+-----------------------------+
+   | Header | Function| Description                 |
+   +========+=========+=============================+
+   | J2.1   | Ve      |                             |
+   +--------+---------+-----------------------------+
+   | J2.2   | GND     |                             |
+   +--------+---------+-----------------------------+
+   | J2.3   | TBD     |                             |
```

Comment:
Not sure "TBD" is the right term for these?

Reply:
Ok, I think I'll leave it blank instead.